core/src/main/resources/application-test.yaml

```diff
@@ -2,11 +2,39 @@ scheduler:
   baseUrl: http://localhost:8081
   options:
     http-configuration:
-      425-backoff-policy:
+      request-retry-policy:
```

Comment:
BTW, did you add this `request-retry-policy` section with purpose? It's for http client exceptions (network issues).

Reply:
Yes, there is a test for CONNECTION_RESET_BY_PEER GenericHttpClientStandaloneTest.shouldRetryOnFaultAndSucceed.